src/serlio/utils/MayaUtilities.cpp

```diff
@@ -69,8 +71,12 @@ MStatus setEnumOptions(const MObject& node, MFnEnumAttribute& enumAttr,
 }
 
 std::wstring cleanNameForMaya(const std::wstring& name) {
-	auto r = name;
+	std::wstring r = name;
 	replaceAllNotOf(r, MAYA_COMPATIBLE_CHARS);
+
+	if (r.size() > 0 && (DIGIT_CHARS.find(r.front()) != std::wstring::npos))
+		r.insert(0, MAYA_SEPARATOR);
```

Comment:
it could be faster to `return MAYA_SEPARATOR + r` instead of modyfing r (reallocations)... but stuff like that should be measured first.

Reply:
Did a quick offline comparison. It seems both approaches are roughly taking the same amount of time (averaged over a million runs for "_" + "Foobar") 
However just returning with + seems cleaner, so I will switch either way. 

```c++
std::wstring appendPrefix(const std::wstring prefix) {
  std::wstring in = L"Foobar";
  return prefix + in;
}
/* vs */
std::wstring insertPrefix(const std::wstring prefix) {
  std::wstring in = L"Foobar";
  in.insert(0, prefix);
  return in;
}
```

Reply:
interesting, the `if` could also play a role for the compiler optimization... anyways, looks good.